DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Centre/Reports/EvaluationSummaryViewModel.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Centre.Reports
+{
+    using System.Collections.Generic;
+    using System.Linq;
+
+    public class EvaluationSummaryViewModel
+    {
+        public EvaluationSummaryViewModel(string question, Dictionary<string, int> counts, int noResponseCount)
+        {
+            Question = question;
+            var totalResponses = counts.Sum(x => x.Value);
+            if (totalResponses > noResponseCount)
+            {
+                Data = counts
+                    .Select(x => new KeyValuePair<string, float>(x.Key, 100f * x.Value / totalResponses))
+                    .ToDictionary(x => x.Key, x => x.Value);

```

Comment:
Any reason this has to be a dictionary in this case? Might be better as an object or a tuple list so we can improve naming & hence readability.. (and avoid going from dictionary to IEnumerable back to dictionary)🤔